src/structs.jl

```diff
@@ -255,10 +248,10 @@ $(TYPEDEF)
 # Fields
 $(TYPEDFIELDS)
 """
-struct Event
-    name::Maybe{String}
-    trigger::Maybe{Math}
-    event_assignments::Vector{EventAssignment}
+Base.@kwdef struct Event
+    name::Maybe{String} = nothing
```

Comment:
I'm not entirely sure whether you wanted to default this field to `nothing` or not

Reply:
yes, the names are "human IDs" and they can be missing by default. We can eventually add the nothings everywhere else too.